runbooks/key-handover/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum-rpc.publicnode.com"
```

Comment:
Similar comment on not duplicating, I think it's better to reuse code that's been through the PR process before.